setup.py

```diff
@@ -1,43 +1,39 @@
 from setuptools import setup, find_packages
 
-version = '2.1.2.dev0'
+version = "2.1.2.dev0"
 
 setup(
-    name='slc.mailrouter',
+    name="slc.mailrouter",
     version=version,
     description="Framework for handling email in zope",
     long_description=open("README.md").read(),
     # Get more strings from http://pypi.python.org/pypi?%3Aaction=list_classifiers
     classifiers=[
         "Programming Language :: Python",
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.7',
         "Framework :: Plone",
-        ],
-    keywords='smtp2zope email mailin zope',
-    author='Syslab.com GmbH',
-    author_email='info@syslab.com',
-    url='https://github.com/syslabcom/slc.mailrouter',
-    license='GPL',
-    packages=find_packages('src', exclude=['ez_setup']),
-    package_dir = {'' : 'src'},
-    namespace_packages=['slc'],
+        "Framework :: Plone :: 5.0",
+        "Framework :: Plone :: 5.1",
+        "Framework :: Plone :: 5.2",
```

Comment:
I do not think we can claim support for anything other than `"Framework :: Plone :: 5.2"`

Reply:
Fair enough. Changed it.